pandas/tests/io/test_pickle.py

```diff
@@ -501,3 +501,15 @@ def test_read_pickle_with_subclass():
 
     tm.assert_series_equal(result[0], expected[0])
     assert isinstance(result[1], MyTz)
+
+
+def test_read_py27_pickle_with_MultiIndex_column(datapath):
```

Comment:
can this test be moved to just after `test_unicode_decode_error`, better still parameterize `test_unicode_decode_error` with `filename` and `excols`

Reply:
done